dash/dash.py

```diff
@@ -541,12 +541,15 @@ def _relative_url_path(relative_package_path="", namespace=""):
 
             modified = int(os.stat(module_path).st_mtime)
 
-            return "{}_dash-component-suites/{}/{}?v={}&m={}".format(
+            return "{}_dash-component-suites/{}/{}.v{}m{}.{}".format(
                 self.config.requests_pathname_prefix,
                 namespace,
-                relative_package_path,
-                importlib.import_module(namespace).__version__,
+                '.'.join(relative_package_path.split('.')[:-1]),
+                importlib.import_module(namespace).__version__.replace(
+                    '.', '_'
+                ),
                 modified,
+                '.'.join(relative_package_path.split('.')[-1:]),
```

Comment:
new pattern!

Reply:
oh hmm, `.js.map` again...

Reply:
Updated to use a specialized fingerprint building function that should handle these cases consistently in regards to the fingerprint check. https://github.com/plotly/dash/pull/973/files#diff-2b63b39f9669123b2341a57a37913ad8R10